provisioner/template.go

```diff
@@ -733,3 +743,55 @@ func sumQuantities(quantities ...string) (string, error) {
 func awsValidID(id string) string {
 	return strings.Replace(id, ":", "__", -1)
 }
+
+// instanceTypeCPU returns the vCPUs of an instance type provided as k8sresource.Quantity represented as string
+func instanceTypeCPU(context *templateContext, instanceType string) (string, error) {
+	// get the instance type info
+	instanceTypeInfo, err := context.instanceTypes.InstanceInfo(instanceType)
+
+	if err != nil {
+		return "", err
+	}
+
+	cpu := fmt.Sprintf("%v", instanceTypeInfo.VCPU)
```

Comment:
nit: should this just be `%s`?

Reply:
Fixed in the next commit.

Reply:
Actually, the linter started complaining that `int64` shouldn't be put where `String` is expected.

Reply:
Ah, then it's `%d` :)

Reply:
Added!